### ISSUE TITLE
CI-Deployment Setup

### ISSUE THREAD

**Issue by ShawnConn** [attn/attn-cms#32]
Once #27, #28, & #29 has been resolved, the last setup is prepping the Pantheon site for CI/Deployment. 

- CI has been configured but there looks to be an issue with the branch/fork strategy; the CI builds don't work unless the users's fork branch is pushed directly to the `attn` parent repo. This might be just a permission issue (the auth'd devops CI/GitHub account might not have access) or a Pantheon change introduced w/ the CircleCI 2.0 upgrade.
- Lastly, the production deployment scripts will need to be setup. Most this has been done in Publisher 2 (ported via this [branch](https://github.com/ShawnConn/attn-cms/tree/ci-deploy)) but it will need to be updated for the changes since it was last ran.

**Comment by ShawnConn** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [CMS: CI / Deployment](https://trello.com/c/xxpLRPWO/193-cms-ci-deployment)

**Comment by ShawnConn** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [CMS: CI / Deployment](https://trello.com/c/xxpLRPWO/193-cms-ci-deployment)

**Comment by sheelah** [comment]
> CI has been configured but there looks to be an issue with the branch/fork strategy; the CI builds don't work unless the users's fork branch is pushed directly to the attn parent repo. This might be just a permission issue (the auth'd devops CI/GitHub account might not have access) or a Pantheon change introduced w/ the CircleCI 2.0 upgrade.

Worth checking with Pantheon on this?  If it's something on their end, maybe it's an easy fix.

**Comment by ShawnConn** [comment]
Yup, that was the first thing I was going to check when I get to this. The build setup was the same steps as Pub2 so there may have been something that's changed since when I last setup the build.
